testdata/jsight_0.3/ser-to-openapi/req.japi.schema_notation/req.japi.schema_notation.jsight_0.3/jsight_notation_in_directive_body.openapi.todo

```diff
@@ -0,0 +1,127 @@
+{
+  "openapi": "3.0.3",
+  "info": {
+    "title": "",
+    "version": ""
+  },
+  "paths": {
+    "/notation/in/request/body/": {
+      "post": {
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {
+                "type": "object",
+                "properties": {
+                  "name": {
+                    "type": "string"
```

Comment:
There is a mistake here. All primitive types must has an "example" attribute, like:

```
"name": {
  "type": "string",
  "example": "John"
}
```

Reply:
Please, fix all these occurences in this and others pull requests. There are a lot of them.

Reply:
Fixed in this PR.